src/hatti/views/map.cljs

```diff
@@ -329,6 +458,49 @@
                    :on-click (click-fn #(om/update! geofield field))}]
                  (get-label field) [:br]])]]]))))))
 
+(defn map-layer-selector
+  [cursor owner]
+  (reify
+    om/IInitState
+    (init-state [_]
+      {:expanded false
+       :styles ["basic" "streets" "bright" "light" "dark" "satellite"]
```

Comment:
Yes, if we build our custom map styles on Mapbox, then they'll need to be added here and the style link (https://github.com/onaio/hatti/pull/146/files/70df3bdf47fffad7dd1d13a627cc8227eeb608ae#diff-765a06264ee0391b4aede90c1206b186R496) will change slightly. But these are the defaults offered by Mapbox.